packages/gatsby-theme-carbon/src/components/Homepage/homepage.scss

```diff
@@ -4,11 +4,24 @@
   color: $carbon--white-0;
 }
 
-.container--dark a,
-.container--homepage a {
+.container--dark a.#{$prefix}--link,
+.container--dark a.#{$prefix}--link:hover,
+.container--dark a.#{$prefix}--link:active,
+.container--dark a.#{$prefix}--link:active:visited,
+.container--dark a.#{$prefix}--link:active:visited:hover,
+.container--homepage a.#{$prefix}--link,
+.container--homepage a.#{$prefix}--link:hover,
+.container--homepage a.#{$prefix}--link:active,
+.container--homepage a.#{$prefix}--link:active:visited,
+.container--homepage a.#{$prefix}--link:active:visited:hover {
```

Comment:
@vpicone Can I just remove this rules for `container--homepage`? Based on what I've seen here and on the Carbon website, it's the `container--dark` class that actually matters here.

Reply:
Additional note: these selectors need this level of specificity due to the rules they are overriding: .[classname].[classname]:hover, etc
But let me know if you have concerns 👍

Reply:
@jendowns yeah you got it exactly right. The homepage as a dark theme by default so that's the one that matters. Thanks for taking the time to do this!

Reply:
Thanks @vpicone! I pushed an update for this